.gitignore

```diff
@@ -1,5 +1,5 @@
 nimcache
-bin/nwn_*
+bin/*
```

Comment:
This file doesn't belong in the change.

Reply:
That is correct.  Should be fixed in 78f9f259.